### PR TITLE
Build and test the examples

### PR DESCRIPTION
**Pull request by nipunayf**
# Description

The PR improves the daily build, enabling it to support the building and testing of the example programs found in the root directory. Additionally, the PR made the following improvements:

* Added Checkstyle, SpotBugs, and JaCoCo workflows to the Gradle build.
* The directory structure will be adhered to according to the document: https://github.com/ballerina-platform/ballerina-extended-library/blob/main/docs/module-contribution-guide.md
* Updated the vulnerable Java dependencies to resolve the Trivy scan failure.

Related to https://github.com/ballerina-platform/ballerina-extended-library/issues/539

### CODE REVIEW

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

**Review by sachinira — COMMENTED** [review]
Change the file paths pointing to the sample files which can be found on  Module.md. As we have changed the folder structure they will not point to the correct place once the PR is merged. Check if this affects the Samples.md file too

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Execute the following commands to build from the source:
  
 - To build the package:
    ```   
-   bal pack ./cosmosdb
+   bal build ./cosmosdb
```
Comment (sachinira) [inline README.md:35]: As we have changed the folder name, this needs to changed too 
```suggestion
   bal build ./ballerina
```
Same applies for `test` too.

File: examples/admin-operations/user-defined-functions/user_defined_functions.bal [hunk 2]
```diff
@@ -105,7 +105,7 @@ function createRandomUUIDWithoutHyphens() returns string {
     string? stringUUID = java:toString(createRandomUUID());
     if (stringUUID is string) {
         stringUUID = 'string:substring(regex:replaceAll(stringUUID, "-", ""), 1, 4);
-        return stringUUID;
+        return stringUUID is string ? stringUUID : "";
```
[the remark this answers is not in the document]
Comment (NipunaRanasinghe) [reply examples/admin-operations/user-defined-functions/user_defined_functions.bal:108]: Let's apply the same in all the similar places

**Review by NipunaRanasinghe — APPROVED** [review]

**Pull request merged**